OKR.md

```diff
@@ -4,7 +4,23 @@ We try to frame our ongoing work using a process based on quarterly Objectives a
 
 ## 2018 Q4
 
-Writting in Progress
+**Debuggability and Stability**
+- error handling is done correctly and error codes are part of the API - @jacobheun
+- libp2p has a network visualization tool - @vasco-santos
+- js-libp2p has a daemon with a RPC API and a client library - @jacobheun
+- interop tests for go-libp2p and js-libp2p exist for core functionality - @jacobheun
+- a testbed exists and can test 50,000 libp2p nodes (browser and node) - @jacobheun
+- the testbed can run large scale tests for the DHT and Pubsub - @vasco-santos
+
+**Connectivity Magic is better and more reliable**
+- The DHT is enabled by default in js-ipfs and is interoperable with go - @vasco-santos
+- IPNS works over DHT and is interoperable with go - @vasco-santos
+- Nat traversal Spec is ready for js-libp2p to implement - @jacobheun
+
+**Libp2p is seen as the go to p2p networking stack**
+- Usage examples exist for the key modules of libp2p - @jacobheun
```

Comment:
We've had several recent questions around how to use certain modules (dht, circuit-relay), so I thought it would be a good idea to get this on the list

Reply:
I agree! Even when I started creating a new suite of tests for DHT interop, I tried to find examples both on go and js, but there was any.

Reply:
I agree! Even when I started creating a new suite of tests for DHT interop, I tried to find examples both on go and js, but there was any.

Reply:
Regarding the examples, I would like something more. 

I think we should aim to have examples for the key modules, as well as have a general overview to the `README` of several modules, which are really incomplete. I think we can divide this OKR and each one of us could be responsible for the modules which we are lead maintainer. Something like `Usage examples and proper documentation exist for the key modules of libp2p`. What do you think?

Reply:
Regarding the examples, I would like something more. 

I think we should aim to have examples for the key modules, as well as have a general overview to the `README` of several modules, which are really incomplete. I think we can divide this OKR and each one of us could be responsible for the modules which we are lead maintainer. Something like `Usage examples and proper documentation exist for the key modules of libp2p`. What do you think?